### DISCUSSION TITLE
It has been three months since my pack was approved! But I am still not receiving the benefits. How long will it take?

### DISCUSSION THREAD

**Discussion by anik4good** [community/community#146888] [GitHub Education]
### Hi everyone,

It has been three months since my pack was approved! But I am still not receiving the benefits. How long will it take?

**Comment by davevad93** [comment]
Hi @anik4good ,

That's strange, normally the waiting time may vary from a few days to a couple of weeks at most. I suggest you to raise a ticket [here](https://support.github.com/contact/education), select 
"`I am a verified student having trouble redeeming my Global Campus offers`" and ask support why is taking so long.
